scripts/release_cocoapod.sh

```diff
@@ -11,15 +11,13 @@ if [ -z "$TARGET_PODSPEC" ]; then
     exit 1
 fi
 
-echo "Checking current pod registration session"
-pod trunk me
-
 echo "Publishing podspec $TARGET_PODSPEC"
 
-pod trunk push $TARGET_PODSPEC \
+time pod trunk push $TARGET_PODSPEC \
```

Comment:
"time" meant for development or deployment? :)

Reply:
thanks, this is for both. Added here to measure the overall trunk push runtime.  Won't have any effect on the released pod itself.